packages/react/src/components/pickers/BasePicker.types.ts

```diff
@@ -235,6 +235,12 @@ export interface IBasePickerProps<T> extends React.Props<any> {
    * Theme provided by styled() function.
    */
   theme?: ITheme;
+
+  /**
+   *  Refers to the iconName for the remove item action
+   *  @defaultvalue 'Cancel'
+   */
+  iconName?: string;
```

Comment:
It'd be better if it were a full IIconProps object so someone could specify any icon props, not just the name. Also, it'd be good to name it to indicate that it only applies to the remove button.

If you change it to IIconProps, see my comment in PeoplePickerItem.tsx for how to implement it.

```suggestion
  removeButtonIconProps?: IIconProps;
```